app/services/provider_interface/hesa_data_export.rb

```diff
@@ -0,0 +1,144 @@
+module ProviderInterface
+  class HesaDataExport
+    def initialize(provider_ids:)
+      @provider_ids = provider_ids
+    end
+
+    def call
+      applications = ApplicationChoice
+        .includes(
+          :course,
+          :provider,
+          :site,
+          course_option: { course: %i[provider accredited_provider] },
+          application_form: %i[candidate application_qualifications],
+        ).where(
+          'candidates.hide_in_reporting' => false,
+          'status' => %w[accepted offer pending_conditions],
```

Comment:
Do we want to include `offer` here — maybe use `ApplicationStateChange::ACCEPTED_STATES`?

Reply:
:+1: @duncanjbrown https://github.com/DFE-Digital/apply-for-teacher-training/pull/3135/commits/14cb7d102cfe29664eaee6492f2bb8c14e6e5b1a